### PR TITLE
✨ Add Ima Leave

### PR DESCRIPTION
**Pull request by shrinerh**

### CODE REVIEW

File: data/foxcraft_dungeon_loot/loot_tables/items/rare/ima_leave.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+        ],
+        [
+          {
+            "text": "you 10,000 meters into the sky",
+            "color": "gray",
+            "italic": false
+          }
+        ],
+        "",
```
Comment (satyrnidae) [inline data/foxcraft_dungeon_loot/loot_tables/items/rare/ima_leave.json]: Multiple perks shouldn't have white space between them, keeps the chance that the lore will just overrun the bottom of the window lower, but good to follow as a convention

File: texturepack/assets/foxcraft_dungeon_loot/models/item/ima_leave.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "parent": "minecraft:item/generated",
+  "textures": {
+    "layer0": "foxcraft_dungeon_loot:item/ima_leave"
+  }
+}
```
Comment (satyrnidae) [inline texturepack/assets/foxcraft_dungeon_loot/models/item/ima_leave.json]: EOF newline

**Review by satyrnidae — CHANGES_REQUESTED** [review]

File: data/foxcraft_dungeon_loot/loot_tables/items/rare/ima_leave.json (hunk 1 above)
Comment (shrinerh) [reply data/foxcraft_dungeon_loot/loot_tables/items/rare/ima_leave.json]: done

File: texturepack/assets/foxcraft_dungeon_loot/models/item/ima_leave.json (hunk 2 above)
Comment (shrinerh) [reply texturepack/assets/foxcraft_dungeon_loot/models/item/ima_leave.json]: done

File: data/foxcraft_dungeon_loot/tags/functions/on_tick.json [hunk 3]
```diff
@@ -27,6 +27,7 @@
     "foxcraft_dungeon_loot:items/d20/on_tick",
     "foxcraft_dungeon_loot:items/titan_hammer/on_tick",
     "foxcraft_dungeon_loot:items/totem_of_mysteria/on_tick",
+    "foxcraft_dungeon_loot:items/ima_leave/on_tick"
```
Comment (satyrnidae) [inline data/foxcraft_dungeon_loot/tags/functions/on_tick.json]: missing a comma here

**Review by satyrnidae — CHANGES_REQUESTED** [review]
missing comma in json

File: data/foxcraft_dungeon_loot/tags/functions/on_tick.json (hunk 3 above)
Comment (shrinerh) [reply data/foxcraft_dungeon_loot/tags/functions/on_tick.json]: fixed

**Review by satyrnidae — APPROVED** [review]
thx thx

**Pull request merged**